### DISCUSSION TITLE
Is there a way to use .mise.toml in the same way as this .envrc?

### DISCUSSION THREAD

**Discussion by aclarknexient** [jdx/mise#1525] [Q&A] [answered]
I have an `.envrc` as follows:

```shell
CURRENT_GIT_BRANCH=$(git branch --show-current)
case "${CURRENT_GIT_BRANCH}" in
  "dev"         ) export ARBITRARY_VAR="foo"      ;;
  "qa"          ) export ARBITRARY_VAR="bar"      ;;
  "main"        ) export ARBITRARY_VAR="fizzbuzz" ;;
  * ) echo "Unknown branch, exiting" ; exit 1 ;;
esac
```

Is there a way to do the same thing but using `.mise.toml`?

Thank you, jdx, for creating mise (rtx), it's fantastic!

**Comment by jdx** [comment, marked as answer]
yes you can use [templates](https://mise.jdx.dev/templates.html)

**Reply by aclarknexient** [reply]
Awesome, *of course* I missed that page while looking at the docs site. Sorry about that!
